src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -56,53 +56,57 @@ public CommandResult(String feedbackToUser) {
         this(feedbackToUser, false, false, false);
     }
 
+    /**
+     * Returns the feedback that should be shown to the user.
+     */
     public String getFeedbackToUser() {
         return feedbackToUser;
     }
 
-    public boolean isResetWidget() {
-        return resetWidget;
-    }
-
+    /**
+     * Returns true if the help window should be shown to the user.
+     */
     public boolean isShowHelp() {
         return showHelp;
     }
 
+    /**
+     * Returns true if the application should exit.
```

Comment:
Would saying something like "Returns true if CommandResult will cause the application to exit" be better? 

tbh the name `isExit` should probably be a little more informative, like `resultIsExit` or `shouldExit()` (to be more in-line with you current comment and standardise with the other `shouldXXX` method names)

Reply:
I think the reason its called isExit is because the commandresult itself is an exit command. Aka there is only one such command that causes the app to exit. Also,  the "Returns true if CommandResult will cause the application to exit" would be less accurate because the commandresult doesn't cause the application to exit, it just carries the message that the application should exit to the mainwindow class, which then implements the exit

Reply:
Alright agreed 👍